make/Makefile.docs

```diff
@@ -0,0 +1,11 @@
+#
+# cOS-toolkit Makefile.docs
+#
+#
+
+
+docs-serve:
```

Comment:
what is the problem for moving the whole makefile here instead of having this fake makefile that calls another makefile?

Reply:
I like that it is having its own Makefile in its own folder until everything is not still "inglobated" (hugo, deps, etc), once we get hugo in the repos most of that is going to disappear and be just 2 scripts